domains/ticketkit.json

```
@@ -0,0 +1,15 @@
+{
+        "site": "https://ticketsx.netlify.app/",
+        "owner": {
+           "username": "blaze-099",
+           "email": "",
+           "discord": "1186691290759434350"
+        },
+    
+        "record": {
+                "A": [
+                    "75.2.60.5",
```

```suggestion
                    "75.2.60.5"
```
This comma shouldn't be here.